packages/common/core-utils/src/events/emitter.ts

```diff
@@ -168,7 +171,7 @@ export class EventEmitter<TListeners extends Listeners<TListeners>>
 			const eventDescription =
 				typeof eventName === "symbol" ? eventName.description : String(eventName.toString());
 
-			throw new UsageError(
+			throw new Error(
```

Comment:
`UsageError` has been replaced with the standard `Error` type because telemetry-utils cannot be added as a dependency in core-utils, due to a layer-check failure.

fyi: @noencke